provider/online_test.go

```diff
@@ -169,7 +169,9 @@ func testBatchSetGetEntity(t *testing.T, store OnlineStore) {
 	if err := batchTable.BatchSet(maxSet); err != nil {
 		t.Fatalf("Failed to set multi entity: %s", err)
 	}
-	for _, item := range maxSet {
+	// We do this for test efficiency purposes, as some bulk writers support large limits, and `Get`ing every
+	// individual one balloons test times.
+	for _, item := range maxSet[:20] {
```

Comment:
I want to bring special attention to this. Firestore has a limit of 500 modifications in a single transaction. If we have strong feelings against this let me know.